src/Fixer/PhpBasic/Feature/ReturnAndArgumentTypesFixer.php

```diff
@@ -46,6 +48,31 @@ public function __construct()
             T_DNUMBER,
             T_LNUMBER,
         ];
+        $this->initializeInflector();
+    }
+
+    private function initializeInflector()
+    {
+        if (class_exists('Doctrine\Inflector\InflectorFactory')) {
+            // Doctrine Inflector v2
+            $this->inflector = InflectorFactory::create()->build();
+        } elseif (class_exists('Doctrine\Common\Inflector\Inflector')) {
+            // Doctrine Inflector v1
+            $this->inflector = new Inflector();
+        } else {
+            throw new \RuntimeException('Doctrine Inflector is not available');
+        }
+    }
+
+    private function pluralize($word)
+    {
+        if (method_exists($this->inflector, 'pluralize')) {
+            // Doctrine Inflector v2
+            return $this->inflector->pluralize($word);
+        } else {
+            // Doctrine Inflector v1
+            return Inflector::pluralize($word);
+        }
```

Comment:
Copy-paste code, lets move this to separate class, eg.
```
class Inflector
{
    public function __construct()
    {
         $this->inflector = $this->getInflector();
    }
    
    private function pluralize($word)
    {
         ....
    }
}
```

And usage just `(new Inflector())->pluralize()`